package/upgrade/upgrade_node.sh

```diff
@@ -494,6 +494,15 @@ spec:
 EOF
 }
 
+# issue 2707 is where this default was set, https://github.com/harvester/harvester/issues/2707 
+set_max_pods() {
+  local cfg=$HOST_DIR/etc/rancher/rancherd/config.yaml
```

Comment:
rancherd config is only loaded in bootstrap time. We need to update RKE2 config before upgrade RKE2.